examples/storybook/src/stories/QueryRuleCustomData.stories.ts

```diff
@@ -99,33 +112,4 @@ storiesOf('QueryRuleCustomData', module)
         },
       },
     }),
-  }))
-  .add('picking first with transform', () => ({
-    component: wrapWithHits({
-      ...moviesConfig,
-      template: `
-      <p>
-        Type <q>music</q> and a banner will appear.
-      </p>
-      <ais-query-rule-custom-data [transformItems]="transformItems">
-      </ais-query-rule-custom-data>`,
-      methods: {
-        transformItems: items => [items[0]],
-      },
-    }),
-  }))
-  .add('keeping only banners with transform', () => ({
-    component: wrapWithHits({
-      ...moviesConfig,
-      template: `
-      <p>
-        Type <q>not a banner</q> and nothing will appear.
```

Comment:
I thought this was a good use case for transformItems

Reply:
Yes, not exactly sure why I removed them. I added them back!